### ISSUE TITLE
login issue

### ISSUE THREAD

**Issue by TwilightTime7734** [tedchou12/webull#227]
No matter how I try to login I get this msg:
>>> from webull import webull
>>> wb = webull()
>>> wb.login('doug******@gmail.com','********')
{'msg': 'Wrong format for mobile phone number', 'traceId': 'ae591773e8cc404495c8b00e77ecd451', 'code': 'phone.illegal'}
>>> (edited the gmail.com)
I try the phone way and it says:
>>> wb = webull()
>>> wb.login('+1-707*******','*********')
{'extInfo': {'desensitizedPhone': '+1-707*******', 'phone': '+1-707********', 'accountType': '1', 'desensitizedEmail': 'dou****@gmail.com', 'equipmentCheck': 'send', 'email': 'dou****@gmail.com', 'account': '+1-707********'}, 'firstTimeOfThird': False}
>>>
Any idea what I am doing wrong?
Much appreciated.
Doug

**Comment by hfacey** [comment]
for that email login, looks like you tried `@gail.com` instead of `@gmail.com`
also, it may be an MFA issue?

**Comment by TwilightTime7734** [comment]
Thank you for replying, and so quickly. I rechecked and typed in the @gmail.com with the same results. Wierd thing is I get a message in my gmail that a new device logged on and then it gives me a security code. Any clue how to send the security code?
And is there any more documentation around that might explain how to implement the access_token and/or refresh_token. I'm trying real hard to figure this out.
Thanks

**Comment by hfacey** [comment]
Checkout our wiki pages:
[MFA explanation](https://github.com/tedchou12/webull/wiki/MFA-&-Security)
[Login workaround](https://github.com/tedchou12/webull/wiki/Workaround-for-Login) (if needed)

**Comment by TwilightTime7734** [comment]
I found a solution, what I ended up doing was installing the mobile apk then logging on. I then went to the file in /data/data/org.dayup.stocks/shared_pref/user_123....uuid...889.xml
That file contained all the data needed ie. refreshToken, accessToken, uuid, tokenExpireTime and more.
I wan't to thank you @tedchou12 for your help and your YouTube videos. Much Help.

I'm going to now write my java API's, when i'm done if anyone wants it just let me know.
